felix/dataplane/linux/bpf_ep_mgr.go

```diff
@@ -1120,6 +1122,17 @@ func (m *bpfEndpointManager) syncIfaceCounters() error {
 		return fmt.Errorf("cannot list interfaces: %w", err)
 	}
 
+	// Update Generic Receive Offload [GRO] if configured.
+	var config = map[string]bool{
+		ethtool.EthtoolRxGRO: false,
+	}
+	for _, iface := range m.bpfDisableGROForIfacesMap {
```

Comment:
If we put it here, it would (a) ask for renaming the `syncIfaceCounters` to something more generic and (b) we want to run it in the loop over the interfaces. Ideally, `bpfDisableGROForIfacesMap` would be a regexp so that we do not need to name eth0, eth1, enPxyz, eni123 etc. Typically, where this is a problem, we would like to use the dataiface regexp to apply this to HEPs

Reply:
Thanks @tomastigera - Done: renamed function to more generic `syncIfaceProperties` and aligned `bpfDisableGROForIfaces` as regexp to `BPFDataIfacePattern` as above